crates/bevy_math/src/bounding/bounded2d/mod.rs

```diff
@@ -277,6 +287,19 @@ mod aabb2d_tests {
         assert!(!shrunk.contains(&a));
     }
 
+    #[test]
+    fn scale_around_center() {
+        let a = Aabb2d {
+            min: Vec2::new(-1., -1.),
+            max: Vec2::new(1., 1.),
+        };
+        let scaled = a.scale_around_center(Vec2::new(2., 2.));
+        assert!((scaled.min - Vec2::new(-2., -2.)).length() < std::f32::EPSILON);
+        assert!((scaled.max - Vec2::new(2., 2.)).length() < std::f32::EPSILON);
```

Comment:
Maybe a matter of preference, but you can use `splat` when every element is the same
```suggestion
        assert!((scaled.min - Vec2::splat(-2..)).length() < std::f32::EPSILON);
        assert!((scaled.max - Vec2::splat(2.)).length() < std::f32::EPSILON);
```
same for 3D

Reply:
I followed the style choices for the above tests here, since they choose to not use `splat`. There's a mix of both though overall, and I think I also prefer `splat`.